sub-packages/bionemo-llm/src/bionemo/llm/model/biobert/model.py

```diff
@@ -453,6 +453,8 @@ def forward(
         output = {"token_logits": logits, "binary_logits": binary_logits}
         if self.include_hiddens:
             output["hidden_states"] = hidden_states.transpose(0, 1).contiguous()  # [s b h] => [b s h]
+        if self.include_hiddens or not self.skip_logits:
+            output["input_ids"] = input_ids
```

Comment:
why do we want this? just passing back out the inputs?

Does this just make it easier to dump the inference results to a file and include the inputs? If so, I think it's worth coming up with another way to do that, I'm worried this might mess up some other training code or otherwise increase GPU memory allocation by passing around tensors we no longer need

Reply:
I just added an argument to `include_input_ids` optionally per our discussion